### ISSUE TITLE
not safe code block

### ISSUE THREAD

**Issue by Neboer** [dchenk/go-render-quill#6]
let's write some code with code block and render it to HTML
```c
#include<stdio.h>
int main(){
printf("hello world!");
return 0;
}
```
![Imgur](https://imgur.com/download/0A0PsnJ/)
And then ... the rendered html parse <stdio.h> as a HTML tag.

**Comment by dchenk** [comment]
Fixed in 7f02d28af0ce41a78edc6db8ff676d51e74464d8 and 58652f0ae751a918cad2f4ddace73ab4a465be9e

**Comment by Neboer** [comment]
thanks
